src/index.js

```diff
@@ -1,2 +1,53 @@
 /* eslint-disable react/react-in-jsx-scope, react/jsx-filename-extension */
 /* @jsx createElement */
+
+function createElement(tagName, props, ...children) {
+  const element = document.createElement(tagName);
+
+  Object.entries(props || {}).forEach(([key, value]) => {
+    element[key.toLowerCase()] = value;
+  });
+
+  children.flat().forEach((child) => {
+    if (child instanceof Node) {
+      element.appendChild(child);
+      return;
+    }
+    element.appendChild(document.createTextNode(child));
```

Comment:
개인적으로는, 조건문이 끝나는곳 에서도 한칸 띄어주시면 가독성이 좋은거같아요!
```suggestion
    }
    
    element.appendChild(document.createTextNode(child));
```